### PR TITLE
params, core/forkid: schedule cancun fork on goerli

### PR DESCRIPTION
**Pull request by MariusVanDerWijden**
This PR schedules the cancun fork for the goerli testnet as discussed on ACD.
Spec: https://github.com/ethereum/execution-specs/pull/860

We schedule:
goerli at 1705473120

### CODE REVIEW

**Comment by MariusVanDerWijden** [comment]
see https://github.com/ethereum/execution-specs/pull/860

File: core/forkid/forkid_test.go [hunk 1]
```diff
@@ -91,8 +91,10 @@ func TestCreation(t *testing.T) {
 				{5000000, 0, ID{Hash: checksumToBytes(0x757a1c47), Next: 5062605}},             // Last Berlin block
 				{5062605, 0, ID{Hash: checksumToBytes(0xB8C6299D), Next: 1678832736}},          // First London block
 				{6000000, 1678832735, ID{Hash: checksumToBytes(0xB8C6299D), Next: 1678832736}}, // Last London block
-				{6000001, 1678832736, ID{Hash: checksumToBytes(0xf9843abf), Next: 0}},          // First Shanghai block
-				{6500000, 2678832736, ID{Hash: checksumToBytes(0xf9843abf), Next: 0}},          // Future Shanghai block
+				{6000001, 1678832736, ID{Hash: checksumToBytes(0xf9843abf), Next: 1705473120}}, // First Shanghai block
+				{6500002, 1705473119, ID{Hash: checksumToBytes(0xf9843abf), Next: 1705473120}}, // Last Shanghai block
+				{6500003, 1705473120, ID{Hash: checksumToBytes(0x70cc14e2), Next: 0}},          // First Cancun block
+				{6500003, 2705473120, ID{Hash: checksumToBytes(0x70cc14e2), Next: 0}},          // Future Shanghai block
```
Comment (Rjected) [inline core/forkid/forkid_test.go:97]:
```suggestion
				{6500003, 2705473120, ID{Hash: checksumToBytes(0x70cc14e2), Next: 0}},          // Future Cancun block
```

**Review by Rjected — COMMENTED** [review]
🫡🚀

File: core/forkid/forkid_test.go (hunk 1 above)
Comment (MariusVanDerWijden) [reply core/forkid/forkid_test.go]: thx

**Review by rjl493456442 — APPROVED** [review]

**Comment by holiman** [comment]
Maybe let's not merge this until jan 4th, next ACD, to get it confirmed. The fork date is jan 18th or so, according to the grapevine (a.k.a @lightclient )

**Review by holiman — APPROVED** [review]
LGTM

**Review by lightclient — APPROVED** [review]

**Comment by cshintov** [comment]
The release note says 10th Jan.

**Comment by DragonDev1906** [comment]
> The release note says 10th Jan.

This confused me quite a bit when I've read it, too: Releasing a version for support for an (upcoming) new version that is supposed to happen before the release was published:

2024-01-11-102004_1520x372_scrot
https://github.com/ethereum/go-ethereum/releases/tag/v1.13.9

**Comment by karalabe** [comment]
Fixed. Sry

**Review by Karma197617 — COMMENTED** [review]
Awesome thank you

**Review by Karma197617 — COMMENTED** [review]
So were good?

**Pull request merged**